pkgs/os-specific/linux/anbox/default.nix

```diff
@@ -45,59 +48,55 @@ in
 
 stdenv.mkDerivation rec {
   pname = "anbox";
-  version = "unstable-2019-11-15";
+  version = "unstable-2020-10-26";
 
   src = fetchFromGitHub {
     owner = pname;
     repo = pname;
-    rev = "0a49ae08f76de7f886a3dbed4422711c2fa39d10";
-    sha256 = "09l56nv9cnyhykclfmvam6bkcxlamwbql6nrz9n022553w92hkjf";
+    fetchSubmodules = true;
+    rev = "7f1bfaf90de1f808d4ce8f49ef050120c5c8d1ef";
+    sha256 = "6qmgKdXI5pHXWecQuCCQTjoG+M+gdmTFrrOb7mC3jJw=";
   };
 
   nativeBuildInputs = [
+    cmake
+    gtest
     makeWrapper
+    python3
   ];
 
   buildInputs = [
-    cmake pkgconfig dbus boost libcap gtest systemd mesa glib
-    SDL2 SDL2_image protobuf protobufc properties-cpp lxc python
+    boost
+    dbus
+    elfutils
+    glib
+    glm
+    libbfd
+    libcap
+    libdwarf
     libGL
+    lxc
+    mesa
+    pkgconfig
+    properties-cpp
+    protobuf
+    protobufc
+    SDL2
+    SDL2_image
+    systemd
   ];
```

Comment:
I am not sure about the build inputs. I do not think they were separated correctly before, but I think some of them should still be moved to nativeBuildInputs. Are the snapcraft build-packages/stage-packages dependencies listed here https://github.com/anbox/anbox/blob/7f1bfaf90de1f808d4ce8f49ef050120c5c8d1ef/snap/snapcraft.yaml#L262-L310 analogous to nativeBuildInputs/buildInputs? Is there a way to sandbox dependencies of a Nix package at run time to test this or do you just have to know? I see at least pkgconfig I am sure is a nativeBuildInput.

Reply:
According to their reference for snapcarft.yaml yes but I am not sure if this translates 1:1 to NixOS.

https://snapcraft.io/docs/snapcraft-yaml-reference

Reply:
Please set `strictDeps = true;`, that should help identifying which are `nativeBuildInputs`.